src/cloudinary.js

```diff
@@ -730,6 +736,48 @@ class Cloudinary {
     return Transformation.new(this.config()).fromOptions(options).setParent(this);
   }
 
+
+  /**
+   * @description This function will will append a TransparentVideo element to the htmlElContainer passed to it.
+   *              TransparentVideo can either be an HTML Video tag, or an HTML Canvas Tag.
+   * @param {HTMLElement} htmlElContainer
+   * @param {string} publicId
+   * @param {object} options The {@link TransparentVideoOptions} options to apply - Extends TransformationOptions
+   *                 options.playsinline    - HTML Video Tag's native playsinline - passed to video element.
+   *                 options.poster         - HTML Video Tag's native poster - passed to video element.
+   *                 options.loop           - HTML Video Tag's native loop - passed to video element.
+   *                 options?.externalLibraries = { [key: string]: string} - map of external libraries to be loaded
+   * @return {Promise<HTMLElement | {status:string, message:string}>}
+   */
+  injectTransparentVideoElement(htmlElContainer, publicId, options = {}) {
+    return new Promise((resolveMainPromise, rejectMainPromise) => {
```

Comment:
Minor - why not removing the 'main' from the reject/resolveMainPromise?
It made sense when you had promise hell at some point, but now it's quite clean.

Reply:
Agrered, changed it.